### PR TITLE
Added files to run integration tests with Stackable CI infrastructure

### PR DESCRIPTION
**Pull request by backstreetkiwi**
## Description
The integration tests are now run in the new CI (https://ci.stackable.tech). The files in the .ci/ folder contain the cluster definition and test script.

## Review Checklist
- [x] Code contains useful comments

### CODE REVIEW

**Review by maltesander — DISMISSED** [review]
LGTM.

**Comment by backstreetkiwi** [comment]
Sorry, @maltesander, I just added the node labels because I hope this makes the test successful...

File: .ci/integration-tests/aws-centos-8/test.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/stackable.sh testdriver-1 -i /.cluster/key 'kubectl label node main-1.stackable.test node=1'
+/stackable.sh testdriver-1 -i /.cluster/key 'kubectl label node main-2.stackable.test node=2'
+/stackable.sh testdriver-1 -i /.cluster/key 'kubectl label node main-3.stackable.test node=3'
```
Comment (maltesander) [inline .ci/integration-tests/aws-centos-8/test.sh:3]: Spark never needed that? I think it was only zookeeper?

**Review by maltesander — CHANGES_REQUESTED** [review]

File: .ci/integration-tests/aws-centos-8/test.sh (hunk 1 above)
Comment (backstreetkiwi) [reply .ci/integration-tests/aws-centos-8/test.sh:3]: It was just a desperate try, I copied it from ZooKeeper ;-)

Comment (backstreetkiwi) [reply .ci/integration-tests/aws-centos-8/test.sh:3]: I will revert that change.

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request merged**